src/test/java/com/hocs/test/glue/decs/BaseStepDefs.java

```diff
@@ -80,12 +73,12 @@ public void iOpenCloseTheAccordionSection(String accordionLabel) {
 
     @And("I select 'Save changes'")
     public void iSelectSaveChanges() {
-        safeClickOn(saveChangesRadioButton);
+        selectSpecificRadioButton("Save changes");
```

Comment:
This will need to be checked, I vaguely remember one of the helper methods not working properly when the header wasn't specified when it was available, but it could've been the checkbox one so it'd be worth investigating